docs/reference/api.md

```diff
@@ -1870,7 +1868,6 @@ _Appears in:_
 | `label` _string_ | Label defines the physical port label you can see on the actual switch |  |  |
 | `group` _string_ | If port isn't directly manageable, group defines the group it belongs to, exclusive with profile |  |  |
 | `profile` _string_ | If port is directly configurable, profile defines the profile it belongs to, exclusive with group |  |  |
```

Comment:
It's generated file and management flag on port is unrelated to management connections